rclcpp_examples/test/test_executables_example.py.in

```diff
@@ -30,7 +31,20 @@ def test_executable():
         else:
             exit_handler = ignore_exit_handler
             exit_on_match = False
-        handler = create_handler(name, launch_descriptor, output_file, exit_on_match=exit_on_match)
+
+        filtered_prefixes = [
+            b'pid', b'rc',
+        ]
+
+        if rmw_implementation == 'rmw_connext_cpp' \
+            or rmw_implementation == 'rmw_connext_dynamic_cpp':
```

Comment:
You could also do `if rmw_implementation in ['rmw_connext_cpp', 'rmw_connext_dynamic_cpp']:` if you wanted to limit it to those two.